tests/test_main.py

```diff
@@ -1,2 +1,33 @@
-def test_hello_world():
-    print("hi")
+import pandas as pd
+import pytest
+import tarfile
+
+from video2dataset.main import video2dataset
+
+
+def test_e2e():
+    url_list = "tests/test_files/test_webvid.csv"
+    destination = "tests/dataset"
```

Comment:
I advise using a temporary dir using tempdir lib
makes sure things are clean when testing  and after